packages/tools/guiEditor/src/main.scss

```diff
@@ -83,7 +88,55 @@
         }
     }
     .toolbar-icon {
-        width: 40px;
+        width: 20px;
+        height: 40px;
+    }
+}
+
+#toolbarGrabExpanded {
+    background: #333333;
+    grid-row: 1 / span 3;
+    grid-column: 2;
+    height: 100%;
+    overflow-y: auto;
+
+    .blackLine {
+        height: 16px;
+        background: black;
+        position: relative;
+    }
+    .arrow {
+        color: white;
+        position: absolute;
+    }
+    .toolbar-content-sub1 {
+        column-count: 2;
+    }
+    .toolbar-label {
+        color: white;
+        font-size: 18px;
+        display: grid;
+        align-items: center;
+        cursor: pointer;
+        user-select: none;
+        grid-template-columns: 1fr 20px;
+        grid-template-rows: 100%;
+        position: relative;
+
+        &.active {
+            font-weight: bold;
+            font-size: 20px;
+        }
+
+        &:hover {
+            background: var(--selectionGrey);
+            .sub-items {
+                display: block;
+            }
+        }
+    }
+    .toolbar-icon {
+        width: 20px;

```

Comment:
with the right classes' definition, you can keep the id to be the same, and just add the difference between the two.
I mean that `toolbarGrabExpanded` is not needed. you can keep `toolbarGrab`, and add the class `expanded` or `toolbar-expanded` with the definition of the changes only.